### ISSUE TITLE
Missing dependency: jsonapi_swagger_helpers

### ISSUE THREAD

**Issue by wadetandy** [jsonapi-suite/jsonapi_suite#5]
Tries to require the swagger helpers without either checking failing gracefully or having it as a dependency: https://github.com/jsonapi-suite/jsonapi_suite/blob/master/lib/jsonapi_suite.rb#L7

**Comment by bastianl** [comment]
Getting this as well, in the new `0.5.2` release:
```
Using jsonapi_suite 0.5.2 (was 0.5.1)
Bundle updated!
➜ rails s                                                                                                                                
/usr/local/lib/ruby/gems/2.4.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require': cannot load such file -- jsonapi_swagger_helpers (LoadError)
```

**Comment by richmolj** [comment]
This is fixed in master, though I won't be able to push a new gem until later today.

In the meantime adding

`gem "jsonapi_swagger_helpers"` to your `Gemfile` should work.

**Comment by richmolj** [comment]
Fixed in 0.5.3
